recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/arm-trusted-firmware/files/0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch

```diff
@@ -1,7 +1,7 @@
-From 0bdaa54e0f72443930b066251ff276d282c7a2b4 Mon Sep 17 00:00:00 2001
+From b199ba985199f0dd56553d1e3fd2a1570317a57a Mon Sep 17 00:00:00 2001
```

Comment:
Please split this commit into two patches in order to separate changes for fixing ARM TF build and (0001-rcar-gen3-arm-trusted-firmware-Allow-to-add-more-bui.patch) and aligning existing commit with new ARM TF version (0001-rcar-Use-UART-instead-of-Secure-DRAM-area-for-loggin.patch)

Reply:
Done